### DISCUSSION TITLE
Unable to sign up using GitHub

### DISCUSSION THREAD

**Discussion by ltwsdc** [supabase/supabase#29535] [General]
I am trying to sign up, using GitHub and I am getting a timeout on the authentication page. 

![image](https://github.com/user-attachments/assets/eb849900-0f51-4a3d-ac90-c3be88109747)

**Comment by silentworks** [comment]
Try in a different browser. If that still doesn't work maybe your ISP is blocking the domain so I would suggest trying a VPN to see if it works.
